app/src/main/java/net/aiscope/gdd_app/ui/mask/MaskActivity.kt

```diff
@@ -175,10 +176,14 @@ class MaskActivity : AppCompatActivity(), MaskView, CaptureFlow {
     }
 
     private fun refreshBrushDrawableColor() {
+        // Retrieve the dot behind the draw icon and set to currently selected colour
         val stageBtnDrawable = binding.drawBtn.compoundDrawables[0] as LayerDrawable
```

Comment:
Do we still need to tint the draw button now that we tint the stages one? Seems redundant, and like a visual cacophony.

Reply:
Dot is gone now